pkgs/os-specific/linux/lvm2/default.nix

```diff
@@ -64,30 +87,39 @@ stdenv.mkDerivation {
 
   doCheck = false; # requires root
 
+  makeFlags = stdenv.lib.optionals (udev != null) [
+    "SYSTEMD_GENERATOR_DIR=$(out)/lib/systemd/system-generators"
```

Comment:
Use placeholder instead of shell variable for $out?

Reply:
I think using shell variables is preferred over placeholder:
https://discourse.nixos.org/t/what-is-the-difference-between-placeholder-out-and-out/4862/2

Reply:
We use placeholder to not have to manually assemble `cmakeFlags`, but in places where using shell variables work, let's make use of it.